index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <div>
+            <h3 class="logo">My Logo</h3>
+        </div>
+        <div class="menu">
+            <img src="./assets/icons/Union.png" alt="Menu Icon">
+        </div>
```

Comment:
- [ ] Kindly make sure that the hamburger icon is wrapped in `nav tag`. Replace div with nav tag.